packages/genesys-spark-components/src/components/beta/gux-tooltip-base/gux-tooltip-base.scss

```diff
@@ -44,6 +44,12 @@
     color: var(--gse-ui-tooltip-light-iconColor);
   }
 
+  ::slotted(*) {
+    width: 100%;
+    overflow-wrap: break-word;
```

Comment:
Was this tested with asian languages? I've had weird things when trying to to wrap Japanese characters as each character is considered a word.

Reply:
I tested it with Japanese. It looks good in the new beta tooltip, but the stable tooltip still overflows the tooltip container.

Reply:
It works in the stable one too but you can't style text nodes so you have to slot an element. That is why I am moving over to the named slot as that does not allow you to slot a text node.